codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/http/HttpBindingGenerator.kt

```diff
@@ -181,10 +181,10 @@ class HttpBindingGenerator(
         httpMessageType: HttpMessageType = HttpMessageType.RESPONSE
     ): RuntimeType {
```

Comment:
Unrelated to this PR, but we could fix it here:

The docblock for this function reads (GitHub won't let me add the comment since it's above the modified hunk):

```
/**
* Generate a function to deserialize `[binding]` from the response payload.
*/
```

We should update it to say "request/response payload".

Reply:
I can fix this. Regarding the commit message, it is kinda backward, sorry about it. I'll fix.

Reply:
I will use the right commit message when I squash all of this.